modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/Button.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import React from 'react';
+
+export default function Button({
+	componentId: _componentId,
+	cssClass: _cssClass,
```

Comment:
Shouldn't we pass `cssClass` as `className` to `<ClayButton` ?

Reply:
I'm pretty sure `className` does get passed in `...otherProps` 

I based this on [`DropdownMenu.js`](https://github.com/liferay/liferay-portal/blob/c379f2a0f2204cf2ded7688e367ef69d72919485/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js)

Reply:
`className` attribute in clay tags [were marked as deprecated for 7.3](https://github.com/liferay/liferay-portal/blob/f6c547babecad34dd09819cf90ce5647b2c5230f/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/internal/servlet/taglib/BaseContainerTag.java#L74) and it's no longer in the `tld`.

All custom classes should be added via `cssClass` attribute, that (I think)  will not be passed inside `otherProps` if it's defined as a prop itself.

In `DropdownMenu.js` you can see in [line 56](https://github.com/liferay/liferay-portal/blob/c379f2a0f2204cf2ded7688e367ef69d72919485/modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/DropdownMenu.js#L56) that `cssClass` is being passed to the trigger button (it was decided that it will be applied here instead to the root dropdown element due to the current usage of that tag)

Reply:
> className attribute in clay tags were marked as deprecated for 7.3 and it's no longer in the tld.

Yes but `className` is used for React components 

I don't understand what you want me to change here @carloslancha  but I have no problem doing it if it's required

Reply:
@carloslancha should be fixed in [84f1d263205fd40d2aca3483b2a4a59238dbc38d](https://github.com/liferay-frontend/liferay-portal/pull/375/commits/84f1d263205fd40d2aca3483b2a4a59238dbc38d) but please tell me what you think before I re-run the **ci** tests.